NtupleProducer/test/datasets_Legacy2016.txt

```diff
@@ -0,0 +1,96 @@
+=== DATA_2016 ===
+/SingleElectron/Run2016B-17Jul2018_ver1-v1/MINIAOD
+/SingleElectron/Run2016B-17Jul2018_ver2-v1/MINIAOD
+/SingleElectron/Run2016C-17Jul2018-v1/MINIAOD
+/SingleElectron/Run2016D-17Jul2018-v1/MINIAOD
+/SingleElectron/Run2016E-17Jul2018-v1/MINIAOD
+/SingleElectron/Run2016F-17Jul2018-v1/MINIAOD
+/SingleElectron/Run2016G-17Jul2018-v1/MINIAOD
+/SingleElectron/Run2016H-17Jul2018_ver2-v1/MINIAOD
+/SingleElectron/Run2016H-17Jul2018_ver3-v1/MINIAOD
+
+/SingleMuon/Run2016B-17Jul2018_ver1-v1/MINIAOD
+/SingleMuon/Run2016B-17Jul2018_ver2-v1/MINIAOD
+/SingleMuon/Run2016C-17Jul2018-v1/MINIAOD
+/SingleMuon/Run2016D-17Jul2018-v1/MINIAOD
+/SingleMuon/Run2016E-17Jul2018-v1/MINIAOD
+/SingleMuon/Run2016F-17Jul2018-v1/MINIAOD
+/SingleMuon/Run2016G-17Jul2018-v1/MINIAOD
+/SingleMuon/Run2016H-17Jul2018-v1/MINIAOD
+
+/Tau/Run2016B-17Jul2018_ver1-v1/MINIAOD
+/Tau/Run2016B-17Jul2018_ver1-v2/MINIAOD
```

Comment:
-> /Tau/Run2016B-17Jul2018_ver2-v1/MINIAOD

Reply:
should this replace both /Tau/Run2016B or just the second one?

Reply:
only the second one